Roles/Crewmate/CopyCat.cs

```diff
@@ -322,7 +323,7 @@ public static bool OnCheckMurder(PlayerControl pc, PlayerControl tpc)
                     SabotageMaster.Add(pc.PlayerId);
                     break;
                 case CustomRoles.Admirer:
-                    Admirer.Add(pc.PlayerId);
+                    CustomRoles.Admirer.GetRoleClass()?.Add(pc.PlayerId);
```

Comment:
yeah so gurge told me this is wrong 😅, anyways ignore copycat I made it rewritten here #713